test/sanity/code-smell/docs-build.py

```diff
@@ -37,7 +37,6 @@ def main():
     }
 
     ignore_codes = [
-        'literal-block-lex-error',
```

Comment:
I'm delighted that this change gets rid of those `cannot lex literal block` errors!

Reply:
Me too. This is probably a side-effect for the extended YAML lexer, which probably now accepts some things which aren't formally correct YAML. I'm still wondering which exact pieces produced these errors; it would be interesting to check why they had a problem with the old YAML lexer...